### PR TITLE
SKALE-3756 Add needs_reload to sChainRecord, reload sChains after SSL…

### PR DESCRIPTION
**Pull request by dmytrotkk**
… upload

### CODE REVIEW

File: web/models/schain.py [hunk 1]
```diff
@@ -129,6 +136,13 @@ def set_schains_first_run():
     query.execute()
 
 
+def set_schains_needs_reload():
```
Comment (badrogger) [inline web/models/schain.py]: Typo: schains_need

File: web/migrations.py [hunk 2]
```diff
@@ -57,6 +58,13 @@ def add_repair_mode_field(db, migrator):
     )
 
 
+def add_needs_reload_field(db, migrator):
```
Comment (badrogger) [inline web/migrations.py:61]: We can add migration into test.migrations_test.

File: core/schains/creator.py [hunk 3]
```diff
@@ -169,14 +169,21 @@ def monitor_schain(skale, node_info, schain, ecdsa_sgx_key_name):
...
-
     checks = SChainChecks(name, node_id, rotation_id=rotation_id)
 
-    if not checks.exit_code_ok:
-        mode = MonitorMode.SYNC
+    logger.debug(f'sChain record: {SChainRecord.to_dict(schain_record)}')
+
+    if schain_record.needs_reload:
+        logger.warning(f'Going to reload {schain["name"]}')
+        remove_schain_container(schain["name"])
```
Comment (badrogger) [inline core/schains/creator.py:178]: Can we use MonitorMode.REGULAR here?

**Review by badrogger — CHANGES_REQUESTED** [review]

File: web/models/schain.py (hunk 1 above)
Comment (dmytrotkk) [reply web/models/schain.py]: fixed

**Review by badrogger — APPROVED** [review]

**Pull request merged**